README.md

```diff
@@ -1,50 +1,38 @@
+# GraphRAG 
 [![Dockerhub](https://img.shields.io/docker/pulls/falkordb/falkordb?label=Docker)](https://hub.docker.com/r/falkordb/falkordb/)
 [![pypi](https://badge.fury.io/py/graphrag_sdk.svg)](https://pypi.org/project/graphrag_sdk/)
 [![Discord](https://img.shields.io/discord/1146782921294884966?style=flat-square)](https://discord.gg/6M4QwDXn2w)
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](code_of_conduct.md) 
+<p align="center">
+  <img alt="FalkorDB GraphRAG-SDK README Banner" src="images/FalkorDB GraphRAG-SDK README Banner.png" width="1500">
+</p>
 
-# GraphRAG-SDK
+### Build fast and accurate GenAI apps with GraphRAG SDK at scale
 
-[![Try Free](https://img.shields.io/badge/Try%20Free-FalkorDB%20Cloud-FF8101?labelColor=FDE900&style=for-the-badge&link=https://app.falkordb.cloud)](https://app.falkordb.cloud)
+Documentation available at: [LINK](https://docs.falkordb.com/)

```

Comment:
could be confusing, this is mainly FalkorDB core DB documentation, I think the reader expectation is to find `GraphRAG-SDK` documentation

Reply:
So for now we can remove it?